packages/manager/modules/manager-components/src/resource-selector/component.js

```diff
@@ -0,0 +1,14 @@
+import template from './template.html';
+
+export default {
+  bindings: {
+    currencySymbol: '@?',
+    price: '<?',
```

Comment:
accept a price object instead of price and currencySymbol

Reply:
The price has to be a string that also contains the currency. The currencySymbol has been removed

Reply:
ok then is it possible to use string binding?